contrib/opencensus-context/examples/explicit_threading.py

```diff
@@ -0,0 +1,38 @@
+from threading import Thread
+from opencensus.common.runtime_context import RuntimeContext
+
+RuntimeContext.register_slot('correlation_context', lambda: dict())
+
+
+def withcc(fn):
```

Comment:
This is to demo how to propagate the context to an explicitly created thread (only the user would know whether it is appropriate to propagate the context) using the `withcc` approach.

Current demo is far from ideal, we probably need to provide a `RuntimeContext.snapshot()` and `RuntimeContext.apply()` mechanism, which would loop through the slots and take copy. (which relates to @c24's question on whether we need to copy by ref or by value)

Reply:
Added RuntimeContext.with_current_context.